docs/Getting Started/Connecting to an Infotainment System/index.md

```diff
@@ -114,6 +114,11 @@ Once you know the IP address, you need to set the websocket `endpoint` and app `
  }
 ```
 
+!!! NOTE
+The `<app name>` value in `"nicknames"` must match the `APP_NAME` value used in [Integration Basics](Getting Started/Integration Basics - Java#implementing-sdl-manager) when implementing the SDL manager.
+!!!
+!@
```

Comment:
I think this needs to be deleted, If you view the guides, on the bottom of the page you can see that something got messed up.
```suggestion

```
![Screen Shot 2022-05-31 at 3 40 02 PM](https://user-images.githubusercontent.com/15617304/171271374-fabba9c0-1f07-4ff0-ab3d-4696aa726f97.png)

Reply:
The hanging closing tag has been removed